int32 timestamp
uint32 seq_rpy
uint32 seq_xyz

uint32 consume 

int32 age

geometry_msgs/Twist controlInfo

float32 roll
float32 pitch
float32 yaw

float32 vx
float32 vy
float32 altd